plio/views.py

```diff
@@ -373,13 +373,16 @@ def metrics(self, request, uuid):
         # else fetch the object
         plio = self.get_object()
 
-        num_survey_questions = Question.objects.filter(item__plio=plio.id, survey=True)
+        # question-based metrics
```

Comment:
why has this comment been moved?

Reply:
Else there would be an error on line 379 ```survey_questions = questions.filter(survey=True)``` as the questions variable won't be defined

Reply:
how will a comment cause errors?